Basic-Video-Chat-With-ForegroundServices-Java/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon May 04 10:15:13 CEST 2020
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-bin.zip
```

Comment:
This versions is very old, isn't it?

Reply:
same. A bit old. I will bump up for all samples